include/ebpf_nethooks.h

```diff
@@ -56,6 +56,7 @@ bind_hook_t(bind_md_t* context);
 
 #define BPF_SOCK_ADDR_VERDICT_REJECT 0
 #define BPF_SOCK_ADDR_VERDICT_PROCEED 1
```

Comment:
Shouldn't this then be 
```suggestion
#define BPF_SOCK_ADDR_VERDICT_PROCEED_SOFT 1
```
?

("No because Linux has PF_SOCK_ADDR_VERDICT_PROCEED" would be a good answer if true.
"No because it was already this way" would be a bad answer since you could simply add
`#define BPF_SOCK_ADDR_VERDICT_PROCEED BPF_SOCK_ADDR_VERDICT_PROCEED_SOFT` for backwards compat, but since we're making breaking changes anyway, even that isn't really needed.)

Reply:
Sure, I can add a "_SOFT" here. Looking at https://docs.ebpf.io/linux/program-type/BPF_PROG_TYPE_CGROUP_SOCK_ADDR/, I think Linux just returns 0/1. I think this change should be backwards compatible, so we'll need the #define BPF_SOCK_ADDR_VERDICT_PROCEED BPF_SOCK_ADDR_VERDICT_PROCEED_SOFT.

Reply:
Why do you need backwards compatibility given we're making breaking changes in this release anyway?  I don't follow.

Reply:
Ah I see, I didn't realize we were making breaking changes in general in this release.

Reply:
@andrwli : if both Dave and Anurag are OK changing BPF_SOCK_ADDR_VERDICT_PROCEED --> BPF_SOCK_ADDR_VERDICT_PROCEED_SOFT, lets do it.

Reply:
Created #4676. Please let me know if there's anything I should change about the issue